src/main/resources/checkstyle/checkstyle.xml

```diff
@@ -0,0 +1,300 @@
+<?xml version="1.0"?>
```

Comment:
this shouldn't reside in src/main/resource, since we don't want to include this file with the app - it only has build time purpose. Generally it should be placed in the root dir or a subdir thats not part of src/

Reply:
Ok, that makes sense.